### PR TITLE
Replay

### PR DESCRIPTION
**Pull request by Gabbendorf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# The Hangman Game
+
+<img src='public/game_over.jpg' alt='hangman' width='450' align='middle'/>
```
Comment (RabeaGleissner) [inline README.md:3]: 😂

File: app.rb [hunk 2]
```diff
@@ -49,10 +49,22 @@ def update_won_games_count
       session[:games_won] += 1
     end
 
+    def keep_same_win_counts(game)
```
Comment (RabeaGleissner) [inline app.rb:52]: I assume this method is there to fix the issue that it increments the win count every time you go to the /end-of-game route... but it seems kind of like a hack! It's not very intuitive to understand when you see that you're subtracting 1 from the number of games won. Is there a different way to solve this problem?

**Review by RabeaGleissner — COMMENTED** [review]
Looks pretty good! Not much to comment on. I can't find anything to criticise hehehe

File: README.md (hunk 1 above)
Comment (Gabbendorf) [reply README.md:3]: 😆

File: app.rb (hunk 2 above)
Comment (Gabbendorf) [reply app.rb:52]: I couldn't find another solution 😭 
I hope to come up with something else while styling the app...

**Pull request merged**